core/src/sumeragi/main_loop.rs

```diff
@@ -911,7 +927,21 @@ pub(crate) fn run<F: FaultInjection>(
             },
             Role::ObservingPeer => match message {
                 Some(Message::BlockCreated(BlockCreated { block })) => {
-                    if let Some(block) = vote_for_block(sumeragi, &state, block) {
+                    if current_view_change_index >= 1 {
```

Comment:
why don't we instead move the node role from observing to validating?

Reply:
That would be more complicated in my opinion. We are trying to move closer to the whitepaper. So some things make less sense because we don't do everything that the paper describes yet. But I think it's useful to adhere to it and then make a change like this once we are closer.

Reply:
sure, though I don't see this proposal conflicting with the whitepaper

Reply:
if nobody has anything to say, feel free to close this conversation before merging